### DISCUSSION TITLE
Which Setting In Mastercomfig Disables Custom Player Ghosts In Graveyard?

### DISCUSSION THREAD

**Discussion by Jgr9** [mastercoms/tf2cfg#686] [Q&A]
Which setting in Mastercomfig Disables (or Enables) the Custom **Player Ghosts** in Graveyard - arena_lumberyard_event ?

Model Quality?
Characters?
(Props?)
Something Else?

(Unless something else is going wonky) It seems to be between Medium High and Medium presets.  On Intel Integrated CPU Graphics from 2016 there's a pretty large shift in performance between Medium and Medium High.  But I'd like to still have the custom player ghosts on Graveyard enabled.  So I'm trying to use Medium with tweaks instead of all the additions of Medium High.

Or is there a better, more specific command to do only that? (custom player ghost model enablement on maps that have them)

**Comment by treacherousfiend** [comment]
I'm not sure this is a comfig issue, since i don't think its something that comfig can mess with in any way, The map itself tells the game what model to load and to my knowledge there's no console command that controls that. on top of that, someone made an issue post on the source 1 issue repo about them not working sometimes (https://github.com/ValveSoftware/Source-1-Games/issues/4431), and it seems like maybe its related to a similar issue where the custom hud textures break on Graveyard? (https://github.com/ValveSoftware/Source-1-Games/issues/4214)

**Reply by Jgr9** [reply]
I wouldn't rule that out.  There's plenty of stuff like that when loading in TF2 maps. (the white lines and squares mostly on the jungle themed maps are particularly fun).

Just not sure (yet) if this (graveyard player ghost models) is in relation to Mastercomfig settings, or not.  It does show the default ghost model, fwiw, so... it doesn't feel like some sort of corruption? idk

**Comment by Jgr9** [comment]
Ok, Yeah, I guess that is what was happening.  I tried Medium locally and it used the custom ghosts.
